app/mailers/support_mailer.rb

```diff
@@ -8,4 +8,13 @@ def confirm_sign_in(support_user, device:)
       subject: 'New sign in to Support for Apply for teacher training',
     )
   end
+
+  def fallback_sign_in_email(support_user, token)
+    @magic_link = support_interface_authenticate_with_token_url(token: token)
```

Comment:
Can we pass in the token and use the path helper in the view?

Reply:
Fixed!